doc/services/sensing/index.rst

```diff
@@ -0,0 +1,282 @@
+   sensing_open_sensor_by_dt(DEVICE_DT_GET(DT_CHOSEN(zephyr_sensing_base_accel)), cb_list, handle);
+
+This method is useful and easy use for some simple Application which just want to access specific
+  sensor(s).
+
+
+``Sensor type`` follows the
+    `HID standard sensor types definition <https://usb.org/sites/default/files/hutrr39b_0.pdf>`_.
+
+    See :zephyr_file:`include/zephyr/sensing/sensing_sensor_types.h`
+
+Sensor Instance Handler
+=========================
+
+Clients using a ``void*`` type handler to handle a opened sensor
+instance, and all subsequent operations on this sensor instance need use this handler,
+such as set configurations, read sensor sample data, etc.
+
+For a sensor instance, could have two kinds of clients:
+``Application clients`` and ``Sensor clients``.
+
+``Application clients`` can use :c:func:`sensing_open_sensor` to open a sensor instance
+and get it's handler.
+
+For ``Sensor clients``, there is no open API for opening a reporter, because the client-report
+relationship is built at the sensor's registration stage with devicetree.
+
+The ``Sensing Subsystem`` will auto open and create ``handlers`` for client sensor
+to it's reporter sensors.
+``Sensor clients`` can get it's reporters' handlers via :c:func:`sensing_sensor_get_reporters`.
+
+.. image:: images/sensor_top.png
+   :align: center
+   :alt: Sensor Reporting Topology.
+
+.. note::
+   Sensors inside the Sensing Subsystem, the reporting relationship between them are all auto
+   generated by Sensing Subsystem according devicetree definitions, handlers between client sensor
+   and reporter sensors are auto created.
+   Application(s) need call sensing_sensor_open() APIs to explicitly open the sensor instance.
+
+Sensor Sample Value
+==================================
+
+* Data Structure
+
+  Each sensor sample value defined as a common ``header`` + ``readings[]`` data structure:
+
+  .. code-block:: c
+
+      struct sensing_sensor_value_xxx {
+         struct sensing_sensor_value_header header;
+         struct data {
+            uint32_t timestamp_delta;
+            union {
+                          ...
+            };
+         } readings[1];
+      };
+
+  The ``header`` definition:
+
+  .. code-block:: c
+
+      struct sensing_sensor_value_header {
+         /** base timestamp of this data readings, unit is micro seconds */
+         uint64_t base_timestamp;
+         /** count of this data readings */
+         uint16_t reading_count;
+      };
+
+
+* Time Stamp
+
+  Time stamp unit in sensing subsystem is ``micro seconds``.
+
+  The ``header`` defined a **base_timestamp**, and
+  each element in **readings[]** array defined **timestamp_delta**.
+
+  Here use **base_timestamp** (``uint64_t``) and **timestamp_delta** (``uint32_t``) to
+  save memory usage in batching mode.
+
+  The **base_timestamp** is for ``readings[0]``, the **timestamp_delta** is relation
+  to the previous readings.
```

Comment:
Suggestion to to use the text from the CHRE documentation.
```suggestion
  The **timestamp_delta** is in relation to the previous **readings** (or the **base_timestamp**)
  for ``readings[0]``. 
```

Reply:
Updated, thanks a lot!